docs/UserGuide.md

```diff
@@ -156,105 +156,160 @@ Examples:
 * `find Chocolate` followed by `delete 1` deletes the first item in the result of the `find` command.
 * `list Room 2` followed by `delete 3` deletes the third item in the recorded list of items in Room 2.
 
-### Reminding user of expiring items: `reminder`
 
-Shows items that are expiring soon.
+### 3.5. Listing items : `list`
+#### 3.5.1. Listing all items at a specific location
+#### 3.5.2. Listing all items with a specific tag
+#### 3.5.3. Listing all items in StoreMando
 
-Format 1: `reminder numOfDays`
+Shows a list of all items in the inventory.
 
-* `reminder 7` will give you a list containing all items that are expiring within the next 7 days.
-*  Items without expiry date will not be shown.
-*  The numOfDays **must be a positive integer** 1, 2, 3, …​ 
+Format: `list` `list [l/LOCATION]` `list [t/TAG]`
 
-Examples:
-* Given today is 2020-03-10, and the inventory has 2 items: chocolate with expiry date of 2020-03-16 and banana with expiry date of 2020-03-19.
-    * `reminder 7` returns chocolate with expiry date of 2020-03-16 only.
-    * `reminder 10` returns both chocolate with expiry date of 2020-03-16 and banana with expiry date of 2020-03-19.
-    
-Format 2: `reminder numOfDays [TIME_UNIT]`
+* You can view all items in the inventory by typing 'list' without specifying location/tag.
+* The search is case-insensitive. e.g 'room' will match 'Room'.
+* The order of keywords for location search does not matter. e.g. 'Room Living' will match 'Living Room'.
```

Comment:
Just a minor concern, but does the number of spaces matter? e.g. Will 'Room     Living' match 'Living Room'?

Reply:
yes it will still work